.vale.ini

```diff
@@ -0,0 +1,13 @@
+StylesPath = .github/styles
+MinAlertLevel = suggestion
+Vocab = Keptn
+
+[*.{md,mdx}]
+BasedOnStyles = Vale, Google
+
+Vale.Spelling = warning
+
+TokenIgnores = (\w+\(\)), \
```

Comment:
> TokenIgnores allow you to exclude certain inline-level sections of text that don’t have an associated HTML tag that could be used with IgnoredScopes.

https://vale.sh/docs/topics/config/?#tokenignores

@iamrajiv 
Why have we added `TokenIgnores`  in our cause?

Reply:
@vadasambar we need to add some ignore tokens to avoid unnecessary errors like we have mentioned links like `- [node](https://nodejs.org/en/)`, `- [About the Project](#about-the-project)` etc in `.md` and `.mdx` files to ignore these I have added some RegEx rules.